tests/test.c

```
@@ -739,39 +740,10 @@ int test_modules_execution(bool beam, bool skip, int count, char **item)
     }
 #ifndef AVM_NO_JIT
     if (!beam) {
-#if JIT_ARCH_TARGET == JIT_ARCH_X86_64
-        if (chdir("x86_64") != 0) {
-            perror("Error: cannot find x86_64 directory");
+        if (chdir(AVM_JIT_TARGET_ARCH_DIR) != 0) {
+            perror("Error: cannot find " AVM_JIT_TARGET_ARCH_DIR " directory");
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If AVM_JIT_TARGET_ARCH_DIR is a macro then please configure it.

(unknownMacro)